### PR TITLE
Cleanup methods for Map

### PR DESCRIPTION
**Pull request by xNexus-ACS**

### CODE REVIEW

File: Exiled.API/Features/Map.cs [hunk 1]
```diff
@@ -296,6 +296,48 @@ public static GameObject PlaceTantrum(Vector3 position, bool isActive = true)
             return tantrum.gameObject;
         }
 
+        /// <summary>
+        /// Destroy all <see cref="ItemPickupBase"/> objects.
+        /// </summary>
+        public static void CleanAllItems()
+        {
+            var pickups = Object.FindObjectsOfType<ItemPickupBase>();
```
Comment (louis1706) [inline Exiled.API/Features/Map.cs:304]: Pickup.List Exist !??!

File: Exiled.API/Features/Map.cs [hunk 2]
```diff
@@ -296,6 +296,48 @@ public static GameObject PlaceTantrum(Vector3 position, bool isActive = true)
...
+            foreach (Pickup pickup in pickups)
+                pickup.Destroy();
+        }
+
+        /// <summary>
+        /// Destroy all <see cref="BasicRagdoll"/> objects.
+        /// </summary>
+        public static void CleanAllRagdolls()
+        {
+            var basicRagDolls = Object.FindObjectsOfType<BasicRagdoll>();
```
Comment (louis1706) [inline Exiled.API/Features/Map.cs:325]: Ragdoll.List !?!?

File: Exiled.API/Features/Recontainer.cs [hunk 3]
```diff
@@ -39,7 +39,7 @@ public static class Recontainer
         /// <summary>
         /// Gets or sets a value indicating whether the containment zone is open.
         /// </summary>
-        public static bool IsContaimentZoneOpen
+        public static bool IsContainmentZoneOpen
```
Comment (louis1706) [inline Exiled.API/Features/Recontainer.cs:42]: BreakingChange put //TODO : IsContainmentZoneOpen

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Map.cs [hunk 4]
```diff
@@ -296,6 +296,48 @@ public static GameObject PlaceTantrum(Vector3 position, bool isActive = true)
...
+
+            foreach (ItemPickupBase pickup in pickups)
+                pickup.DestroySelf();
+        }
+
+        /// <summary>
+        /// Destroy all the <see cref="Pickup"/> objects from the specified list.
+        /// </summary>
+        /// <param name="pickups">The List of pickups to destroy.</param>
+        public static void CleanAllItems(List<Pickup> pickups)
```
Comment (louis1706) [inline Exiled.API/Features/Map.cs:314]: IEnumerable

File: Exiled.API/Features/Map.cs [hunk 5]
```diff
@@ -296,6 +296,48 @@ public static GameObject PlaceTantrum(Vector3 position, bool isActive = true)
...
+
+            foreach (BasicRagdoll ragDoll in basicRagDolls)
+                NetworkServer.Destroy(ragDoll.gameObject);
+        }
+
+        /// <summary>
+        /// Destroy all <see cref="Ragdoll"/> objects from the specified list.
+        /// </summary>
+        /// <param name="ragDolls">The List of Ragdolls to destroy.</param>
+        public static void CleanAllRagdolls(List<Ragdoll> ragDolls)
```
Comment (louis1706) [inline Exiled.API/Features/Map.cs:335]: IEnumerable

**Review by louis1706 — CHANGES_REQUESTED** [review]

File: Exiled.API/Features/Map.cs (hunk 1 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Map.cs]: fixed

File: Exiled.API/Features/Recontainer.cs (hunk 3 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Recontainer.cs]: fixed

File: Exiled.API/Features/Map.cs (hunk 4 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Map.cs]: same as above XD

File: Exiled.API/Features/Map.cs (hunk 5 above)
Comment (xNexus-ACS) [reply Exiled.API/Features/Map.cs]: :)

File: Exiled.API/Features/Map.cs [hunk 6]
```diff
@@ -296,6 +296,44 @@ public static GameObject PlaceTantrum(Vector3 position, bool isActive = true)
...
+            foreach (Pickup pickup in pickups)
+                pickup.Destroy();
+        }
+
+        /// <summary>
+        /// Destroy all <see cref="BasicRagdoll"/> objects.
+        /// </summary>
+        public static void CleanAllRagdolls()
+        {
+            foreach (Ragdoll ragDoll in Ragdoll.List)
```
Comment (louis1706) [inline Exiled.API/Features/Map.cs:323]: .ToList()

File: Exiled.API/Features/Map.cs [hunk 7]
```diff
@@ -296,6 +296,44 @@ public static GameObject PlaceTantrum(Vector3 position, bool isActive = true)
             return tantrum.gameObject;
         }
 
+        /// <summary>
+        /// Destroy all <see cref="ItemPickupBase"/> objects.
+        /// </summary>
+        public static void CleanAllItems()
+        {
+            foreach (Pickup pickup in Pickup.List)
```
Comment (louis1706) [inline Exiled.API/Features/Map.cs:304]: .ToList()

**Review by louis1706 — CHANGES_REQUESTED** [review]

**Comment by xNexus-ACS** [comment]
also tested

**Review by Misaka-ZeroTwo — APPROVED** [review]

**Review by louis1706 — APPROVED** [review]

**Review by Thundermaker300 — APPROVED** [review]

**Pull request merged**